test/run-tests.js

```diff
@@ -27,4 +27,7 @@ if (version.toLowerCase() === 'all') {
   }
 }
 
-npmRun.execSync(`jest -c '${JSON.stringify(jestConfig)}'`, { stdio: 'inherit' })
+npmRun.execSync(
+  `jest -c '${JSON.stringify(jestConfig)}' ${process.argv.slice(2).join(' ')}`,
```

Comment:
this has actually allowed me to use `--watch` (which is still not ideal because we have to edit copy of tests rather than real test files to benefit from that, but better this now than nothing)